ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantFeature.scala

```diff
@@ -22,9 +22,10 @@ object Features {
     val experimental = features.flatMap(_.experimental)
 
     Features(
-      selfServiceErrorCodes = experimental.flatMap(_.selfServiceErrorCodes).isDefined,
-      userManagement = features.flatMap(_.userManagement).isDefined,
-      staticTime = experimental.flatMap(_.staticTime).isDefined,
+      selfServiceErrorCodes =
+        experimental.flatMap(_.selfServiceErrorCodes.map(_.supported)).contains(true),
+      userManagement = features.flatMap(_.userManagement.map(_.supported)).contains(true),
+      staticTime = experimental.flatMap(_.staticTime.map(_.supported)).contains(true),
       commandDeduplicationFeatures = response.getFeatures.getExperimental.getCommandDeduplication,
     )
```

Comment:
Perhaps using the `.get` prefix here wouldn't be so bad. I think it will use the defaults if the value isn't present.

```scala
    val features = response.getFeatures
    val experimental = features.getExperimental

    Features(
      selfServiceErrorCodes = experimental.getSelfServiceErrorCodes.getSupported == true,
      userManagement = features.getUserManagement.getSupported == true,
      staticTime = experimental.getStaticTime.getSupported == true,
      commandDeduplicationFeatures = experimental.getCommandDeduplication,
    )
```

Reply:
Good point, as now we have the same semantics for None or default. Will change

Reply:
Couldnt you also remove the ` == true` comparisons? They should be idempotent aren't they?